src/redux/reducers/TodoReducer.js

```diff
@@ -1,12 +1,13 @@
 import { actionTypes } from 'redux/constants/ActionTypes';
-
-const { ADD_TODO, SET_ADD_TASK } = actionTypes;
+import { deleteTask } from 'utils/helpers/ReducerHelper';
+const { ADD_TODO, SET_ADD_TASK, DELETE_TODO } = actionTypes;
```

Comment:
Bhai, I don't see any use of `SET_ADD_TASK` here. Is it for any future use?

Reply:
@fza-wtag bhai, it's from step-2. For handling the `isAdding task` state.